test/helpers/WitnetRequestBoardTestHelper.sol

```diff
@@ -51,49 +43,44 @@ contract WitnetRequestsBoardV1 is WitnetRequestsBoardInterface {
   // Event emitted when a result proof is posted
   event PostedResult(address indexed _from, uint256 _id);
 
-  constructor (address _blockRelayAddress, uint8 _repFactor) public {
-    blockRelay = BlockRelayProxy(_blockRelayAddress);
+  constructor (address[] memory _committee, bool _upgradable) public {
     witnet = msg.sender;
+    upgradable = _upgradable;
 
     // Insert an empty request so as to initialize the requests array with length > 0
     DataRequest memory request;
     requests.push(request);
-    repFactor = _repFactor;
+    committee = _committee;
   }
 
   /// @dev Posts a data request into the WRB in expectation that it will be relayed and resolved in Witnet with a total reward that equals to msg.value.
   /// @param _requestAddress The request contract address which includes the request bytecode.
-  /// @param _inclusionReward The amount of value that will be detracted from the transaction value and reserved for rewarding the reporting of the inclusion of the data request.
-  /// @param _tallyReward The amount of value that will be detracted from the transaction value and reserved for rewarding the reporting of the final result (aka tally) of the data request.
   /// @return The unique identifier of the data request.
```

Comment:
unique or sequential?

Reply:
both! :smile: if they are sequential and they do not overflow, they are unique. I will harmonize the docs. Thx